workflows/maker-annotation/.dockstore.yml

```diff
@@ -0,0 +1,7 @@
+version: 1.2
+workflows:
+- name: "ANNOTATION-OF-EUKARYOTIC-GENOMES-WITH-MAKER"
```

Comment:
Any particular reason to have this name in upper case?

Reply:
Not really, except that we named all the covid-19 workflows in uppercase, but there's no technical reason to do it. If there's only a single workflow in the folder we could also just call it `main` ? The only concern is that the name should be reasonably short (if you need to type it manually, or include it as a reference in a paper), and if we add multiple workflows we should know which workflow does what.